builtin/common/misc_helpers.lua

```diff
@@ -550,6 +550,10 @@ if core.gettext then -- for client and mainmenu
 	function fgettext(text, ...)
 		return core.formspec_escape(fgettext_ne(text, ...))
 	end
+
+	function hgettext(text, ...)
```

Comment:
don't forget to add it to `util/updatepo.sh`

Reply:
https://github.com/luanti-org/luanti/pull/16049/commits/1d3713da190942e54638b22bdde5a44c600eae42